packages/server/api/controllers/movies.controller.js

```diff
@@ -50,10 +50,51 @@ const createMovie = async (body) => {
   };
 };
 
+const getFeaturedMovie = async (req, res) => {
+  // this will be replaced with a real database call in the future for last movie
```

Comment:
If the hardcoded is no longed needed then please remove this

Reply:
i guess yes because when i started movies hasn't added